Add option for making the columns the same height

mammuth opened the pull request:
This PR adds a simple checkbox to allow the user to set the height of the columns to the same height. It is done using Flexbox.

Tested with:
- Desktop Chrome
- Desktop Firefox


![equal-height](https://cloud.githubusercontent.com/assets/3121306/25858718/b2599064-34dc-11e7-92f3-55a11a8b9597.png)

Related issue #13

mammuth committed:
Add option for making the columns the same height
Closes #13

SteinRobert commented on layouter/models.py:
```diff
@@ -62,6 +62,7 @@ class ContainerPlugin(CMSPlugin):
 
     margin = models.IntegerField(choices=MARGIN_TYPES, null=False, blank=False, default=MARGIN_TYPES[0][0],
                                  help_text=_('How much margin is needed on the left and right side?'))
+    equal_height = models.BooleanField(_('Make height of the columns equal'), null=False, blank=False, default=False)
```
I think `Align height of columns in this row` would be a better description of what actually happens when the user ticks the box.

SteinRobert committed:
Equal height for columns
========================

- update help text
- add comment for developers because we use flex box

SteinRobert committed:
Equal height for columns
========================

- readme

SteinRobert committed:
Equal height for columns
========================

- readme update

SteinRobert committed:
Equal height for columns
========================

- note about missing migrations
